apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -19,6 +19,12 @@ enum GuildStatus {
   ONBOARDED
 }
 
+enum ImportStatus {
+  Pending
+  Completed
+  Failed
+}
```

Comment:
Oh let's prefer a separate table rather than an enum. That is what we have done for contributions and attestations and it is important to be consistent